images/router/egress/README.md

```diff
@@ -0,0 +1,89 @@
+# OpenShift Egress Router
+
+The OpenShift egress router runs a service that redirects traffic to a
+specified remote server, using a private source IP address which is
+not used for anything else. This can be used to allow pods to talk to
+servers that are set up to only allow access from whitelisted IP
+addresses.
+
+## Deploying the egress router pod
+
+The Pod definition for an egress router will look something like this:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: egress-1
+  labels:
+    name: egress-1
+  annotations:
+    pod.network.openshift.io/assign-macvlan: true
+spec:
+  containers:
+  - name: egress-router
+    image: openshift/origin-egress-router
+    securityContext:
+      privileged: true
+    env:
+    - name: OPENSHIFT_EGRESS_SOURCE
+      value: 192.168.12.99
+    - name: OPENSHIFT_EGRESS_GATEWAY
+      value: 192.168.12.1
+    - name: OPENSHIFT_EGRESS_DESTINATION
+      value: 203.0.113.25
+  nodeSelector:
+    site: springfield-1
+```
+
+The `pod.network.openshift.io/assign-macvlan` annotation tells
+OpenShift to create a macvlan network interface on the primary network
+interface, and then move it into the pod's network namespace before
+starting the egress-router container.
+
+The pod contains a single container, using the
+`openshift/origin-egress-router` image, and that container is run
+privileged so that it can configure the macvlan interface and set up
+iptables rules.
+
+The environment variables tell the egress-router image what addresses
+to use; it will configure the macvlan interface to use
+`OPENSHIFT_EGRESS_SOURCE` as its IP address, with
+`OPENSHIFT_EGRESS_GATEWAY` as its gateway. (This should be the same as
+the default gateway used by the node itself.). It will then set up NAT
+rules so that connections to any TCP or UDP port on the pod's cluster
+IP address will be redirected to the same port on
+`OPENSHIFT_EGRESS_DESTINATION`. In this example, connections to the
+pod will be redirected to **203.0.113.25**, with a source IP address
+of **192.168.12.99**.
```

Comment:
Could you make it a bit clearer where the EGRESS_SOURCE and EGRESS_GATEWAY come from?  eg, they must be reserved by the network administrator for the pod, right?